webpack.config.js

```diff
@@ -136,6 +136,10 @@ var config = {
     ]
   },
   plugins: [
+    new LodashModuleReplacementPlugin({
+      shorthands: true,
```

Comment:
If you can avoid the `shorthands` feature it will save you a considerable amount. It's one of the heaviest features.